### PR TITLE
Get Tweets from 010php account specifically

### PR DESCRIPTION
**Pull request by oldskool**
The Twitter service currently gets the tweets for the account to which the tokens set in the `parameters.yml` file belong. So, if you create tokens on your personal account, it will show your personal tweets on the homepage. This fix makes sure the tweets from the @010php account are fetched, regardless of who owns the tokens.

For documentation on the added parameter, see https://dev.twitter.com/rest/reference/get/statuses/user_timeline

### CODE REVIEW

File: app/config/parameters.yml.dist [hunk 1]
```diff
@@ -16,6 +16,7 @@ parameters:
     twitter_consumer_secret: "ChangeMe"
     twitter_access_token: "ChangeMe"
     twitter_access_token_secret: "ChangeMe"
+    twitter_screen_name: "010php"
```
Comment (reenl) [inline app/config/parameters.yml.dist]: I'd prefer not to configure it unless there is a use case where this is changed based on the environment.

File: src/OhTenPHP/Website/SiteBundle/Services/TwitterService.php [hunk 2]
```diff
@@ -47,9 +42,11 @@ public function __construct(Twitter $client, Pool $cache)
     /**
      * Returns the latest tweets from the defined user timeline.
      *
+     * @param string $screenName The twitter account for which to get the timeline.
+     *
      * @return mixed
      */
-    public function getLatestTweets()
+    public function getLatestTweets($screenName)
```
Comment (reenl) [inline src/OhTenPHP/Website/SiteBundle/Services/TwitterService.php]: If we configure the screen name, I would inject it via the constructor instead, we don't have use cases for different screenNames

**Comment by reenl** [comment]
Yay we got another contributer! Nice job!

**Comment by oldskool** [comment]
@reenl Thanks for your input. I originally had it setup like this: https://github.com/010PHP/010php.nl/pull/45/commits/a52d08381b32497d0009e9d09dce428df8898cca

But I figured that's not as dynamic as setting it through the parameters. But I figure the screen name won't change anytime (soon), so we might as well use that constant instead. I can just rebase the PR to exclude the second commit.

**Comment by reenl** [comment]
@oldskool I get where you are comming from. Lets wait what caroga has to say. For me the last commit is easy to add, but hard to remove, so that is why I would like to postpone it as long as possible.

File: src/OhTenPHP/Website/SiteBundle/Services/TwitterService.php (hunk 2 above)
Comment (Caroga) [inline src/OhTenPHP/Website/SiteBundle/Services/TwitterService.php]: I agree that if this is to be used that we use constructor injection instead.

**Comment by oldskool** [comment]
Closing this in favor of #47

**Pull request closed without merging**